### PR TITLE
deprecated GPL-1.0.xml

### PR DESCRIPTION
**Pull request by jlovejoy**
added isDeprecated tag and note

### CODE REVIEW

**Comment by jlovejoy** [comment]
I did this one as a pull request to make sure it's right and the notes text is okay (so we can then do same for all others as per https://github.com/spdx/license-list-XML/issues/543

still need to notate version for which it was deprecated (v3.0)
@kestewart can you review text in note?

File: src/GPL-1.0.xml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <SPDXLicenseCollection xmlns="http://www.spdx.org/license">
-   <license isOsiApproved="false" licenseId="GPL-1.0"
+   <license isDeprecated="true" isOsiApproved="false" licenseId="GPL-1.0"
             name="GNU General Public License v1.0 only">
       <crossRefs>
          <crossRef>http://www.gnu.org/licenses/old-licenses/gpl-1.0-standalone.html</crossRef>
       </crossRefs>
+      <notes>DEPRECATED: Deprecated in lieu of more explicit license identifier of GPL-1.0-only
```
Comment (goneall) [inline src/GPL-1.0.xml]: Missing </notes> at the end of this line

Comment (goneall) [inline src/GPL-1.0.xml]: Missing </notes>

Comment (jlovejoy) [reply src/GPL-1.0.xml]: added (and deprecatedVersion tag)

**Pull request merged**